falcon/testing/client.py

```diff
@@ -314,8 +319,8 @@ def simulate_request(app, method='GET', path='/', query_string=None,
             may be passed, in which case it will be used as-is.
         json(JSON serializable): A JSON document to serialize as the
             body of the request (default: ``None``). If specified,
-            overrides `body` and the Content-Type header in
-            `headers`.
+            overrides `body` and forces the Content-Type header to
```

Comment:
The `json` docstring clarification should be replicated to other `simulate_*` friends (where applicable).

Furthermore, would it be helpful to explicitly state that by forcing we mean it also takes precedence even over the `content-type` parameter?

Reply:
done